### DISCUSSION TITLE
Visibility at the card level (or possibly entity level) on dashboards

### DISCUSSION THREAD

**Discussion by Tiwing1** [home-assistant/frontend#19098] [Dashboard] [answered]
Add visibility to individual cards rather than just at the "tab" level - would allow much more flexible dashboard build/design and eliminate redundant dashboards for different users. Visibility of course should be maintained at the tab level also!

Even at the element or entity level would be great, but might be too complicated to keep organized. I'm envisioning a menu item added at the card that would allow visibility to be set.
<img width="299" alt="image" src="https://github.com/home-assistant/frontend/assets/48974306/a10591b5-dbe3-4114-955e-75edafda224d">

**Comment by karwosts** [comment, marked as answer]
`conditional` card does have an option to show/hide depending on the logged in user, that might be something like what you're looking for.

**Comment by Tiwing1** [comment]
I'm switching to that - had never noticed user in conditional before, thank you.

In my opinion it's a relatively complicated work-around - ideally would be a simple menu item, but this will definitely work for the meantime. Thank you!!
